KarhooSDK/Service/Payment/KarhooPaymentService.swift

```diff
@@ -39,13 +39,11 @@ final class KarhooPaymentService: PaymentService {
 
     func initialisePaymentSDK(paymentSDKTokenPayload: PaymentSDKTokenPayload) -> Call<PaymentSDKToken> {
         paymentSDKTokenInteractor.set(payload: paymentSDKTokenPayload)
-
```

Comment:
Why remove these spaces?

Reply:
because it was supposed to be removed like all the other funcs in that class